automatic_push/README.md

```diff
@@ -25,3 +25,15 @@ This package contains python scripts to process IETF and openconfig YANG files:
     iana-if-type@2022-08-17.yang
     iana-if-type@2022-08-24.yang
     ```
+  
+- draft_push
+
+  Script automatically pushes new IETF draft yang modules to the GitHub repository: https://github.com/yang-catalog/yang.
+  Old ones are removed and their naming is corrected to ```<name>@<revision>.yang```. New IETF RFC modules are checked 
```

Comment:
I would just add that if `draft_push.py` triggers `rfc_push.py` we should either rename them to `ietf_rfc_push.py` and `ietf_draft_push.py` OR create one script `ietf_push.py`, which will handle both, creating commits for RFC and Drafts and then pushing them to the `yang-catalog/yang`.

Reply:
I moved functionality of ```ietf_rfc_push.py``` into ```ietf_push.py``` and removed the commit of only RFC files because they would be committed and pushed at the end of the script anyways within the cronjob, so there's no reason to distinguish them